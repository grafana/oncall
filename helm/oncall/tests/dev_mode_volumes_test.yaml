suite: dev mode
templates:
  - celery/deployment.yaml
  - engine/deployment.yaml
release:
  name: oncall
tests:
  - it: doesn't add a volume and volume mount when oncall.devMode is disabled
    set:
      oncall:
        devMode: false
    asserts:
      - notExists:
          path: spec.template.spec.containers[0].volumeMounts
      - notExists:
          path: spec.template.spec.initContainers[0].volumeMounts
      - notExists:
          path: spec.template.spec.volumes
