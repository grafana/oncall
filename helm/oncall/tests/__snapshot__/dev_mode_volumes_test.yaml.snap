does add a volume and volume mount when oncall.devMode is enabled:
  1: |
    - mountPath: /etc/app
      name: dev-reloaded-engine
  2: |
    - mountPath: /etc/app
      name: dev-reloaded-engine
  3: |
    - mountPath: /etc/app
      name: dev-reloaded-engine
  4: |
    - mountPath: /etc/app
      name: dev-reloaded-engine
  5: |
    - hostPath:
        path: /engine
      name: dev-reloaded-engine
  6: |
    - hostPath:
        path: /engine
      name: dev-reloaded-engine
